test/View/RestfulJsonRendererTest.php

```diff
@@ -460,14 +460,14 @@ public function testRendersAttributeAsPartOfPaginatedCollectionResource()
         $test       = $this->renderer->render($model);
         $test       = json_decode($test);
 
-        $this->assertInstanceof('stdClass', $test, var_export($test, 1));
-        $this->assertObjectHasAttribute('count', $test, var_export($test, 1));
+        $this->assertInstanceof('stdClass', $test, var_export($test, true));
```

Comment:
Ah, yes, and these changes as well. All `var_export` calls in the tests were using `1` as the second param, but by having strict types defined on these test files (I removed those from the `src` files for now) PHP was complaining that `1` is not a `bool` so changed these to `true`.